### PR TITLE
[project-base] promocode flow in cart is covered by acceptance tests

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| ...
|New feature| No 
|[BC breaks](/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| #818  
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/upgrade/UPGRADE-unreleased.md [hunk 1]
```diff
@@ -229,6 +229,67 @@ for instance:
     - `OrderStatusDataFactory`
     - `ParameterDataFactory`
     - `UnitDataFactory`
+- (*low priority*) if you want to test flow of coupon code in cart page, implement new acceptance tests ([#825](https://github.com/shopsys/shopsys/pull/825))
+    `Tests\ShopBundle\Acceptance\acceptance\CartCest`
```
Comment (vitek-rostislav) [inline docs/upgrade/UPGRADE-unreleased.md]: I am not sure whether it is necessary to provide the diffs here - is not the link to PR sufficient? If you did the fixes based on my comments below, you would need to add more diffs here...

Comment (vitek-rostislav) [reply docs/upgrade/UPGRADE-unreleased.md]: more on that, should this be in UPGRADE notes at all? Maybe changelog is sufficient...:thinking:

Comment (boris-brtan) [reply docs/upgrade/UPGRADE-unreleased.md]: i think that tests for functional discount codes are essential, but who knows.

File: project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Front/CartPage.php [hunk 2]
```diff
@@ -130,28 +130,36 @@ private function getTotalProductsPriceCell()
...
+        $this->tester->pressKeysByElement($promoCodeField, WebDriverKeys::ENTER);
         $this->tester->waitForAjax();
     }
 
-    public function removeCoupon()
+    public function removePromoCode()
     {
-        $removeCouponButton = $this->webDriver->findElement(WebDriverBy::cssSelector('.box-promo-code__added__remove'));
-        $this->tester->clickByElement($removeCouponButton);
+        $removePromoCodeButton = $this->webDriver->findElement(WebDriverBy::cssSelector('.box-promo-code__added__remove'));
```
Comment (vitek-rostislav) [inline project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Front/CartPage.php]: still missing `js-` css class/id here

Comment (boris-brtan) [reply project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Front/CartPage.php]: fixed

**Pull request merged**